### DISCUSSION TITLE
[résolu] 21. Comment choisir son thème

### DISCUSSION THREAD

**Discussion by dNicolle** [cnumr/best-practices-wordpress#11] [Bonnes pratiques] [answered]
Je ne sais pas comment tourner le principe de validation sur cette fiche.
[21. Comment choisir son thème ](https://github.com/cnumr/fiches-wordpress-greenit/blob/main/best-practices-WordPress/fiches/21.%20Comment%20choisir%20son%20th%C3%A8me_.md)

**Comment by hrenaud** [comment]
**Proposition**

| Le nombre ... | est inférieur ou égal à |
| ------------- | :---------------------: |
| de fonctionnalités inutilisées        |            0            |
| de règles d'accessibilités KO        |            0            |
| de test avec l'extension GreenIT au-dessus de C        |            0            |
| de test avec PageSpeed Insight au-dessous de 80% en desktop        |            0            |
| de page builder utilisé        |            0            |

**Reply by dNicolle** [reply]
Re merci ;)

**Comment by florinesueur** [comment, marked as answer]
Cette discussion est en lien avec : https://github.com/cnumr/fiches-wordpress-greenit/discussions/17.
